internal/daemon/api_test.go

```diff
@@ -80,13 +80,18 @@ func (s *apiSuite) TestSysInfo(c *check.C) {
 
 	d := s.daemon(c)
 	d.Version = "42b1"
+	state := d.overlord.State()
+	state.Lock()
+	state.VerifyReboot("ffffffff-ffff-ffff-ffff-ffffffffffff")
+	state.Unlock()
```

Comment:
Just for the record, I've been in touch with the snapd team so that we hopefully change this at some point, as this sort of functionality doesn't make much sense on the state itself, but it should be easy to fix it when the time comes.